content/_data/changelogs/weekly.yml

```diff
@@ -4725,6 +4725,24 @@
           title: changelog for 1.6
       message: |-
         Upgrade SSH CLI Auth Module from 1.5 to 1.7 to remove Trilead SSH references.
+- version: '2.187'
+  date: 2019-07-21
+  changes:
+    - type: rfe
+      pull: 4118
+      message: |-
+        Internal: Update slf4j version from 1.7.25 to 1.7.26
+    - type: rfe
+      pull: 4105
+      message: |-
+        Change the default interval for node monitors (e.g. free disk space) from 60 minutes to 5 minutes).
```

Comment:
It was reverted. The defaults remained intact. 
Please also add a link to https://wiki.jenkins.io/display/JENKINS/Features+controlled+by+system+properties